src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ContentDialog.cs

```diff
@@ -349,7 +349,12 @@ public async Task When_CloseDeferred()
 			async void SUT_Closing(object sender, ContentDialogClosingEventArgs args)
 			{
 				// Closing should only be invoked once.
+				// TODO: !!IMPORTANT!! If this assert failed it will be currently **swallowed**.
+				// It's not going to fail the test nor crash the process.
+				// This is NOT a false positive of the analyzer.
+#pragma warning disable MSTEST0040 // Do not assert inside 'async void' contexts
 				Assert.IsFalse(triggered);
+#pragma warning restore MSTEST0040 // Do not assert inside 'async void' contexts
```

Comment:
FYI @jeromelaban 

It's quite interesting for Uno's case because **generally** if this was a "typical" test app, it would have crashed the whole process. I assume it doesn't crash the process because of NativeDispatcher swallowing the exception (it's likely used via `NativeDispatcherSynchronizationContext`)? So currently such failure will only be logged.

cc @Evangelink